docs/langref/index.rst

```diff
@@ -8,9 +8,9 @@ Introduction to Relay
 
 Relay is a functional, differentiable programming language
 designed to be an expressive intermediate representation for machine
-learning systems. Relay supports closures, control flow, and
-recursion, allowing it to directly represent more complex models
-than can computation graph-based IRs.
+learning systems. Relay supports algebraic data types, closures,
+control flow, and recursion, allowing it to directly represent more
+complex models than can computation graph-based IRs.
```

Comment:
I am not a native speaker, and I don't understand the grammar here..."than can computation"

Reply:
I can rephrase it. It means the same as "than computation graph-based IRs can"